libraries/Zigbee/src/ZigbeeHandlers.cpp

```diff
@@ -148,4 +150,25 @@ static esp_err_t zb_cmd_default_resp_handler(const esp_zb_zcl_cmd_default_resp_m
   return ESP_OK;
 }
 
+static esp_err_t zb_cmd_ias_zone_status_change_handler(const esp_zb_zcl_ias_zone_status_change_notification_message_t *message) {
+  if (!message) {
+    log_e("Empty message");
+  }
```

Comment:
Add return on failure
```suggestion
  if (!message) {
    log_e("Empty message");
    return ESP_FAIL;
  }
```